runtime/platform.c

```diff
@@ -285,7 +286,9 @@ void caml_plat_futex_free(caml_plat_futex* ftx) {
 void caml_plat_futex_wait(caml_plat_futex* ftx,
                           caml_plat_futex_value undesired) {
   while (atomic_load_acquire(&ftx->value) == undesired) {
+    CAML_EV_BEGIN(EV_OS_WAIT);
```

Comment:
`OS_WAIT` seems too general for what's happening here. For example, waiting on `select` or `accept` call would also be "waiting on the OS". Would `FUTEX_WAIT` be more appropriate?